packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -27,16 +27,29 @@
     }
 
     &__title {
+      margin-bottom: $carbon--spacing-05;
       @include carbon--type-style('expressive-heading-03');
+    }
 
-      margin-bottom: $carbon--layout-05;
+    &__title,
+    &__content {
```

Comment:
I would think we need to update the classnames.

Reply:
crap! forgot to do that! good thing you picked up on that!